README.md

```diff
@@ -1,34 +1,56 @@
 # cltdevs meetup manifesto
 
 ## Mission statement
-We grow Charlotte's tech community by bringing people together.
+We are committed to the Charlotte tech community's growth and sustainability by fostering relationships between individuals and organizations, cultivating individual talents and organizations, and discover and build the missing elements of a thriving tech community.
```

Comment:
Please move this to its own PR.  Would love for this conversation to be in a dedicated space.